code/README.md

```diff
@@ -0,0 +1,13 @@
+<!--
```

Comment:
I think that a `LICENSE` file is also missing. It should be a copy of the `LicenseRef-MIT-OA` one.

It may seem unnecessary, but again we should try to make this as stand-alone as possible.
Moreover, as per documentation:
> Most build backends automatically include license files in packages. See your backend’s documentation for more details.